.github/workflows/main.yml

```diff
@@ -113,6 +113,24 @@ jobs:
     - run: cargo test -Z build-std=std --no-run --workspace --target aarch64-apple-tvos --release
     - run: cargo test -Z build-std=std --no-run --workspace --target aarch64-apple-tvos --features parallel
 
+  # This is separate from the matrix above because there is no prebuilt rust-std component for the target.
```

Comment:
Could you create a new matrix to combine this with the aarch64-apple-tvos test above?

Reply:
@Be-ing I updated to have a matrix for all -Z build-std targets. The only ones right now are tvOS targets but any tier 3 targets would fall in this same bucket as well.